tests/cases/fourslash/getOutliningSpans.ts

```diff
@@ -89,12 +89,12 @@
 ////                            module m8[| {
 ////                                module m9[| {
 ////                                    module m10[| {
-////                                        module m11 {
-////                                            module m12 {
-////                                                export interface IFoo {
-////                                                }
-////                                            }
-////                                        }
+////                                        module m11[| {
+////                                            module m12[| {
+////                                                export interface IFoo[| {

```

Comment:
Seems like there should still be a span that exceeds the limit which does NOT get outlined.

Reply:
I agree, but I didn't want to get into another scenario with an unnecessarily large fourslash test. I think 120 would require 50 levels of nested modules to hit the limit. If we commit to setting the limit at 120, I'll go ahead and update the tests.

Reply:
Why do they have to be nested modules? Can't you just have a nested array? e.g. keep going with something like:

```javascript
var x = [[[[[[[[[[[[[[[[[[
[
1,2,3
]
]]]]]]]]]]]]]]]]];
```

Reply:
You're right, that should work fine.